unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/GodMode/Inspector/EntityListAdapter.cs

```diff
@@ -36,6 +36,9 @@ private void Start()
             {
                 Rename(newText);
                 SetTextboxActive(false);
+
+                if (EventSystem.current != null && !EventSystem.current.alreadySelecting)
+                    EventSystem.current?.SetSelectedGameObject(null);
```

Comment:
The null propagation is not really needed since we are checking it.

Reply:
You're right! Thanks! Fixed on https://github.com/decentraland/unity-renderer/pull/635/commits/85e8fc9af0249407d6ec544417c21feb420912a6